panels/info/cc-info-panel.c

```diff
@@ -1939,6 +1876,15 @@ cc_info_panel_init (CcInfoPanel *self)
       g_error_free (error);
     }
 
+  self->priv->client = udisks_client_new_sync (NULL, &error);
```

Comment:
Nitpick: Given that all the initialisation code here is sync, it’s alright to make this call sync as well — but what’s the startup time of the info panel? Is there a noticeable delay when it’s opened? If so, we might want to do some follow-up work to make this async.

Reply:
With my Endless One, I couldn't notice any perceivable increase in initialization time. I'd say, it's safe.